src/Header.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+
+const Header = () => {
+return (
+<header className="App-header">
```

Comment:
Are you using Prettier or a linter like Eslint? Make sure you use one so it catches your indent issues. You want to make sure your code is formatted properly before committing anything to the codebase.

Reply:
https://prettier.io/

Reply:
Solved.